test/e2e/xsite/xsite_test.go

```diff
@@ -99,28 +100,32 @@ func secretSiteName(siteName string) string {
 }
 
 func TestCrossSiteViewInternal(t *testing.T) {
-	testCrossSiteView(t, false, "", ispnv1.CrossSiteExposeTypeClusterIP)
+	testCrossSiteView(t, false, "", ispnv1.CrossSiteExposeTypeClusterIP, 0)
 }
 
 func TestCrossSiteViewKubernetesNodePort(t *testing.T) {
 	// Cross-Site between clusters will need to setup two instances of the Kind for Travis CI
 	// Not be able to test on the separate OCP/OKD instance (probably with AWS/Azure LoadBalancer support only)
-	testCrossSiteView(t, true, config.SchemeTypeKubernetes, ispnv1.CrossSiteExposeTypeNodePort)
+	testCrossSiteView(t, true, config.SchemeTypeKubernetes, ispnv1.CrossSiteExposeTypeNodePort, 0)
 }
 
 func TestCrossSiteViewOpenshiftNodePort(t *testing.T) {
-	testCrossSiteView(t, true, config.SchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeNodePort)
+	testCrossSiteView(t, true, config.SchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeNodePort, 0)
 }
 
 func TestCrossSiteViewKubernetesLoadBalancer(t *testing.T) {
-	testCrossSiteView(t, true, config.SchemeTypeKubernetes, ispnv1.CrossSiteExposeTypeLoadBalancer)
+	testCrossSiteView(t, true, config.SchemeTypeKubernetes, ispnv1.CrossSiteExposeTypeLoadBalancer, 0)
 }
 
 func TestCrossSiteViewOpenshiftLoadBalancer(t *testing.T) {
-	testCrossSiteView(t, true, config.SchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer)
+	testCrossSiteView(t, true, config.SchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer, 0)
 }
 
-func testCrossSiteView(t *testing.T, isMultiCluster bool, schemeType string, exposeType ispnv1.CrossSiteExposeType) {
+func TestCrossSiteViewLoadBalancerWithPort(t *testing.T) {
+	testCrossSiteView(t, true, config.SchemeTypeOpenShift, ispnv1.CrossSiteExposeTypeLoadBalancer, 1443)
```

Comment:
@pruivo , @ryanemerson please pay attention that CI and my local machine unable to test services on privileged ports (less than 1000). Probably it's an MetalLB limitations. but need to be tested on Azure or AWS LoadBalancer

Reply:
I did a test yesterday in my laptop with port 443 and it worked fine (kind 0.11.1 + MetalLB v0.10.2)
for testing purposes, we just need the port to be different from 7900

Reply:
Thanks for confirmation. Will try with newer MetalLB version